source/libsass.html.haml

```diff
@@ -37,11 +37,17 @@ title: libSass
   %li#go
     :markdown
       ### Go
-      [Wellington](https://github.com/wellington/wellington) is an extension to libsass that adds spriting and is available on brew:
-
-      `brew install wellington`
-
-      There are also three other libsass wrappers in go: [gosass](https://github.com/moovweb/gosass), [go-sass](https://github.com/SamWhited/go-sass) and [go_sass](https://github.com/suapapa/go_sass) which have not been updated in a while.
+      [go-libsass](https://github.com/wellington/go-libsass) has the most active
+      GoLang wrapper.  [gosass](https://github.com/moovweb/gosass) is another
+      libsass wrapper.
+      
+      [Wellington](http://getwt.io/) is an extension to libsass that adds
+      spriting.  It is available on brew: `brew install wellington`
+
+      [C6](https://github.com/c9s/c6) is a SASS 3.2 compatible implementation
```

Comment:
Ah, I misread the JS entry "The only pure JavaScript implementation is sass.js" which technically libsass compiled into JS JIT via emscripten.  So kinda still libsass.

Well obviously Ruby-sass & libsass aren't the only compilers in town anymore.  They should be listed somewhere...  What's the best approach?  Rename this page to "alternative Sass compilers" & rework it?